4.0.1/README.md

```diff
@@ -115,3 +115,25 @@ docker run --name geonetwork -d -p 8080:8080 \
 -e GEONETWORK_DB_NAME=mydbname \
 geonetwork:4.0.1
 ```
+
+
+## Monitoring
+
+A composition is also available for monitoring metrics and logs 
+for the webserver and the database. 
+
+In Kibana go to `Manage space` and create a `catalogue-monitoring` space.
+This space will be populated with default dashboards by metricbeat and filebeat.
+
+Use the following to start metricbeat and filebeat:
+
+```shell script
+docker-compose -f docker-compose.yml -f docker-compose.monitoring.yml up --build
+```
```

Comment:
With a fresh copy of the repository, trying to launch the compo using the command, I have an issue with filebeat / metricbeat containers:

```
filebeat_1       | 2020-12-16T10:07:18.283Z	ERROR	instance/beat.go:951	Exiting: error connecting to Kibana: fail to get the Kibana version: HTTP GET request to http://www:80/geonetwork/dashboards/s/catalogue-monitor/api/status fails: parsing kibana response: unexpected end of JSON input. Response: .
filebeat_1       | Exiting: error connecting to Kibana: fail to get the Kibana version: HTTP GET request to http://www:80/geonetwork/dashboards/s/catalogue-monitor/api/status fails: parsing kibana response: unexpected end of JSON input. Response: .
[...]
metricbeat_1     | 2020-12-16T10:07:18.438Z	ERROR	instance/beat.go:951	Exiting: error connecting to Kibana: fail to get the Kibana version: HTTP GET request to http://www:80/geonetwork/dashboards/s/catalogue-monitor/api/status fails: parsing kibana response: unexpected end of JSON input. Response: .
metricbeat_1     | Exiting: error connecting to Kibana: fail to get the Kibana version: HTTP GET request to http://www:80/geonetwork/dashboards/s/catalogue-monitor/api/status fails: parsing kibana response: unexpected end of JSON input. Response: .
401_filebeat_1 exited with code 1
401_metricbeat_1 exited with code 1

```

Reply:
Seems that the containers are expecting JSON output, but the response is 0-bytes here:

```
% curl 'http://localhost:8080/geonetwork/dashboards/s/catalogue-monitor/api/status' -i
HTTP/1.1 200 OK
Content-Length: 0
Server: Jetty(9.4.35.v20201120)

```

Reply:
Because you have to have Kibana to first start no?
So if you restart filebeat and metricbeat once all others are up it is ok ?

Reply:
Ok, this is because the default geonetwork image used in the compo should ship with the following commit:
https://github.com/geonetwork/core-geonetwork/commit/ca209c19dd26642db4d34f52d869fa202a90ae89

so that we can use a basic-authentication to access kibana.

One option is to build a 4.0.x geonetwork, then copy the war in the `4.0.1` directory then use:
```
docker-compose -f docker-compose.yml -f docker-compose.monitoring.yml -f docker-compose.dev.yml up --build
```

Also since the geonetwork service bootstrap is not immediate, we might need to wait and launch the monitoring services after having launched geonetwork & configured a "catalogue-monitor" space in kibana. To sum this up:

```
docker-compose -f docker-compose.yml -f docker-compose.dev.yml up --build
[ gn / kibana space creation by hand in the admin ui ]
docker-compose -f docker-compose.yml -f docker-compose.monitoring.yml up -d --build
```